components/filter/PlantFilterForm.js

```diff
@@ -0,0 +1,93 @@
+import styled from "styled-components";
+import { useRef } from "react";
+
+export default function PlantFilterForm({ onSubmit, onClear }) {
+  const form = useRef(null);
+
+  function handleClearFilter() {
+    form.current.reset();
+    onClear();
+  }
```

Comment:
instead, please use `<button type="reset">`, this will handle the resetting for you. You can still pass the onClear function as the onClick handler to it.